config/header-snippets/types_legacy.h.in

```diff
@@ -85,9 +85,14 @@ typedef signed long long duk_int64_t;
     (defined(DUK_F_MINGW) || defined(DUK_F_MSVC))
 /* Both MinGW and MSVC have a 64-bit type. */
 #define DUK_F_HAVE_64BIT
+#if defined(DUK_F_MSVC)
+typedef unsigned __int64 duk_uint64_t;
```

Comment:
MSVC does not have 64-bit longs, they are 32-bit even in 64-bit mode. The most compatible type in that compiler is __int64 (long long is also available, but requires more recent versions).